packages/crypto/src/bip39.ts

```diff
@@ -1,8 +1,5 @@
-import { fromHex, toHex, toUtf8 } from "@cosmjs/encoding";
+import { fromHex, toHex } from "@cosmjs/encoding";
 import * as bip39 from "bip39";
-import { pbkdf2 } from "pbkdf2";
-import * as unorm from "unorm";
```

Comment:
Could you remove the now unused dependencies `unorm` and `pbkdf2` from `packages/crypto/package.json`?

Reply:
Also `@types/pbkdf2` and `@types/unorm` 🔥